packages/fe-container/src/hooks/useAppData/useAppData.ts

```diff
@@ -27,13 +28,34 @@ const navigateToFirstPlugin = (configuration: Configuration) => {
   }
 }
 
+const setCssProperty = (propertyName: string, propertyValue: string | undefined) => {
+  const propertyColor = tinycolor(propertyValue)
+  const propertyToApply = propertyColor.isValid() ? propertyColor.toString() : getComputedStyle(document.documentElement).getPropertyValue(propertyName)
+  document.documentElement.style.setProperty(propertyName, propertyToApply)
+}
+
+const applyColorAlpha = (propertyValue: string | undefined, alphaValue: number) => {
+  const propertyColor = tinycolor(propertyValue)
+  if (propertyColor.isValid()) {
+    propertyColor.setAlpha(alphaValue)
+  }
+  return propertyColor.toRgbString()
```

Comment:
If `propertyColor.isValid()` is `false`, this line works or throws? Maybe add a test and eventually a fallback